lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -162,5 +163,20 @@ sub parent_group_overview {
     $self->group_overview('JobGroupParents', 'main/parent_group_overview');
 }
 
+sub changelog {
+    my ($self) = @_;
+
+    my $changelog;
+    if (open(my $changelog_file, '<', '/usr/share/openqa/public/ChangeLog')) {
```

Comment:
this is too specific to our packages, this needs to be configured (by our package). And if no changelog file is defined, the menu entry shouldn't be there.